### ISSUE TITLE
projects/core/howto/threading.rst should be properly formatted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6944]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#6944|
|Type|task|
|Created|2014-02-03 17:25:25Z|
|Branch|https://github.com/twisted/twisted/tree/threading-docs-format-6944|

As with [#6943](https://github.com/twisted/twisted/issues/6943), this is updating the rST sources to be correctly formatted.

<details><summary>Searchable metadata</summary>

```
trac-id__6944 6944
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_threading_docs_format_6944 branches/threading-docs-format-6944
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1391448325000000 1391448325000000
changetime__1391685713000000 1391685713000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [41537]) Branching to threading-docs-format-6944.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl removed owner|
|-|-|

Putting this up for review.

Most of the changes are formatting. I did update one instance of "threadpool" to "thread pool" and changed callFromThread to use the short name to be consistent.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_3"></a>|@hynek commented|
|-|-|

1.33: Lone colons are basically always wrong.  Typographically  you want endashes, thus `--`. In this case I’d just split it up into two sentences for simplicity and clarity.
 1. 35: :api:`twisted.internet.threads <twisted.internet.threads>` looks rather redundant.
 1. 55: double twisted.internet.reactor & twisted.internet.threads again
 1. 74: I think we’re doing at least two blank lines after code?
 1. 90: same
 1. 117: double twisted.python.threadpool.ThreadPool
 
Please fix and merge. :)

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_4"></a>|@hynek commented|
|-|-|

FWIW, 4 & 5 are optional. I hope [#6905](https://github.com/twisted/twisted/issues/6905) will clarify on that.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

> Typographically you want endashes

Hm.  I think en dashes are basically always wrong. :)  (I wonder how much this varies between languages.)  I think you actually want a period and a new sentence.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_6"></a>|@hynek commented|
|-|-|

Hm, my source would be: http://www.smashingmagazine.com/2011/08/15/mind-your-en-and-em-dashes-typographic-etiquette/ and that seems to be the most common thing on the inter webs. Except for pretentious sites that use emdashes without spaces around it. :) But there seems to be various approaches indeed; a bare colon is always wrong, so much is clear. :)

But yeah: two sentences is better here.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

(In [41568]) Merge threading-docs-format-6944: projects/core/howto/threading.rst should be properly formatted

Author: hawkowl
Reviewer: hynek
Fixes: [#6944](https://github.com/twisted/twisted/issues/6944)

Neatens up the rST source of the threading long-form docs.
